doc/modules/ensemble.rst

```diff
@@ -128,15 +128,15 @@ Random Forests
 --------------
 
 In random forests (see :class:`RandomForestClassifier` and
-:class:`RandomForestRegressor` classes), each tree in the ensemble is
-built from a sample drawn with replacement (i.e., a bootstrap sample)
-from the training set. In addition, when splitting a node during the
-construction of the tree, the split that is chosen is no longer the
-best split among all features. Instead, the split that is picked is the
-best split among a random subset of the features. As a result of this
-randomness, the bias of the forest usually slightly increases (with
-respect to the bias of a single non-random tree) but, due to averaging,
-its variance also decreases, usually more than compensating for the
+:class:`RandomForestRegressor` classes), each tree in the ensemble is built
+from a sample drawn with replacement (i.e., a bootstrap sample) from the
+training set. In addition, when splitting a node during the construction of the
+tree, the split that is chosen is no longer the best split among all features,
+unless you leave the default parameter ``max_features=’auto’``. Instead, the
+split that is picked is the best split among a random subset of the features.
+As a result of this randomness, the bias of the forest usually slightly
```

Comment:
If this docstring goes into detail about the bias-variance tradeoff, it may be more truthful to emphasize: the variance reduction in random forests, compared to decision trees, is the much more important aspect that dominates over how the bias is different between these two estimators.

Reply:
I provide more details on how the variance is lowered with both the randomness injected in the trees and the tree decisions combining.